pymarsys/connections.py

```diff
@@ -168,4 +168,6 @@ def __init__(self, username, secret, uri=EMARSYS_URI):
                         await response.text()
                     )
                 )
-            return await response.json()
+            if response.headers['Content-Type'] is 'text/json':
```

Comment:
Use `==` to compare strings, not `is`. `is` compares the `id` of the object, not the real value. Strings are optimised in Python, so your string will probably have the same `id` in this situation, but still, `==` would be safer here IMO.

https://stackoverflow.com/questions/2988017/string-comparison-in-python-is-vs